app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -152,15 +157,18 @@ module.exports = React.createClass
           <div key={selects[i].id}>
             {if selects[i].title isnt @props.task.instruction
               <div>{selects[i].title}</div>}
-            <Select
+            {if selects[i].allowCreate
+              SelectComponent = Select.Creatable
+            else
+              SelectComponent = Select}
+            <SelectComponent
               options={options}
               onChange={@onChangeSelect.bind(@, i)}
-              value={@props.annotation.value[i]?.value}
+              value={@state.value[i]?.value}
```

Comment:
so this is driving me crazy - if there's a conditional dropdown, with `allowCreate`, that already has a few options, but a user enters a new option, it adds that new option to the options array in the Select component (child of CreatableSelect), and everything appears to work fine, including specifically the new option is displayed in the input. However, if there aren't already a few options, and a user enters a new option, it gets saved to state and props, but it doesn't show up in the input, and it's not being added to the options prop in the Select component.

In this workflow (`projects/markb-panoptes/focus-testing-project/classify?reload=1&workflow=2737`), the second dropdown in the first task has options, so you should be able to add an option fine. In the second task, the second dropdown does not have any options, but you can add an option and it saves it in props/state/annotation, though it doesn't display in the input.